manifests/s/Samsung/ThemeDesigner/2.0.4/Samsung.ThemeDesigner.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.ThemeDesigner
+PackageVersion: 2.0.4
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+InstallerSwitches:
+  Silent: /S /v"/qn"
+  SilentWithProgress: /S /v"/qn"
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Samsung_Theme_Designer_2.0.4.exe
+  InstallerUrl: https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip

```

Comment:
Hi @erikiorai,

This URL appears to be down. Is the package available from another?

Reply:
Yes its also avelable in web archive https://web.archive.org/web/20160825083554/https://img-developer.samsung.com/contents/cmm/Samsung_Theme_Designer_2.0.4.zip

Reply:
Unfortunately we can't use archive.org links due to their terms of service

Reply:
i found another link https://softpedia-secure-download.com/dl/e6e98cb2762ace0731a0c7b0e73a73aa/65b2844a/100199952/software/mobile/Samsung_Theme_Designer_2.0.4.zip